cime_config/allactive/config_compsets.xml

```diff
@@ -349,16 +271,6 @@
   <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
-<compset>
-  <alias>A_WCYCL1850-DIB</alias>
-  <lname>1850_EAM%AV1C-L_ELM%SPBC_MPASSI%DIB_MPASO%IB_MOSART_SGLC_SWAV</lname>
-</compset>
-
-<compset>
-  <alias>A_WCYCL1850-DIB-ISMF</alias>
-  <lname>1850_EAM%AV1C-L_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
-</compset>
-
 <compset>
   <alias>A_WCYCL1850-DIB_CMIP6</alias>
```

Comment:
Can we rename this one to WCYCL1850-DIB  ?

Reply:
@jonbob , can we remove this one? Look like CRYO1850 is the new equivalent?

Reply:
@wlin7 - which one do you want to remove?

Reply:
I'm asking the cryo people about the older compsets -- the A_WCYCL1850-DIB-ISMF_CMIP6 is almost exactly the same as the new CRYO1850 (only different in specifying spunup ocn/ice IC's)

Reply:
A_WCYCL1850-DIB_CMIP6 and A_WCYCL1850-DIB_CMIP6 are now removed.